proposals/lip-0045.md

```diff
@@ -788,6 +776,16 @@ def sendInternal(sendingAddress: Address,
         "params": EMPTY_BYTES
     }
 
+    # Not possible to send messages to the own chain.
+    if receivingChainID == ownChainAccount.chainID:
+        emitPersistentEvent(
+            module = MODULE_NAME_INTEROPERABILITY,
+            name = EVENT_NAME_CCM_SENT_FAILED,
+            data = {"ccm": ccm, "code": CCM_SENT_FAILED_CODE_INVALID_RECEIVING_CHAIN},
+            topics = []
+        )
+        raise Exception("Sending chain cannot be the receiving chain.")
+
     try:
```

Comment:
I think this try ... except (lines 789-799) should be removed since we now pass the encoded params as input to this function.

Reply:
nice catch! fixed

Reply:
I also removed the now unused constant from the table